assets/questions/rle.js

```diff
@@ -116,9 +126,9 @@ def DecodeEncodedString(word):
   }, {
     instructions: [
       [
-        'Good work! But we need to make sure that the string can accurately ',
-        'be decoded. Think about whether the output of your function will ',
-        'correctly be decoded back to the original string.'
+        'Next, make sure that your method\'s output can accurately be decoded. ',
+        'For each <#x> pair, the decode method will repeat the following character ',
```

Comment:
Not sure what "following" means. Perhaps something like:

For each '<#x>' pair, where # is a number and x is a character, the decode method will repeat the x character # times.